sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentAwardDetails.e2e-spec.ts

```diff
@@ -93,21 +104,19 @@ describe("AssessmentStudentsController(e2e)-getAssessmentAwardDetails", () => {
       {
         offeringIntensity: OfferingIntensity.partTime,
         applicationStatus: ApplicationStatus.Completed,
+        firstDisbursementInitialValues: {
```

Comment:
Should we add the `date_sent` here to ensure it is not returned by API ??

Reply:
I will add it.